### ISSUE TITLE
Unknown Error when training 

### ISSUE THREAD

**Issue by BigFishMaster** [PaddlePaddle/Paddle#8307]
# my environment:
```
GPU: P40, cuda-8, cudnn-v5.1, single device
```
# error message:
```python
Pass 34, batch 0, loss 1.08986544609, acc1 0.7109375, acc5 0.9140625, time 1.02 sec
Pass 34, batch 10, loss 1.09057402611, acc1 0.7421875, acc5 0.90625, time 1.02 sec
Pass 34, batch 20, loss 1.02202308178, acc1 0.734375, acc5 0.8984375, time 1.03 sec
Pass 34, batch 30, loss 1.28327906132, acc1 0.6015625, acc5 0.8828125, time 1.03 sec
Pass 34, batch 40, loss 1.01937317848, acc1 0.7421875, acc5 0.9375, time 1.02 sec
Pass 34, batch 50, loss 0.927886605263, acc1 0.7421875, acc5 0.90625, time 1.02 sec
Pass 34, batch 60, loss 1.03937697411, acc1 0.7421875, acc5 0.890625, time 1.03 sec
Pass 34, batch 70, loss 0.772708773613, acc1 0.8046875, acc5 0.9453125, time 1.02 sec
Pass 34, batch 80, loss 0.7219222188, acc1 0.78125, acc5 0.9609375, time 1.03 sec
*** Aborted at 1518151974 (unix time) try "date -d @1518151974" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGSEGV (@0xa802) received by PID 2581 (TID 0x7f512c7a0700) from PID 43010; stack trace: ***
    @     0x7f512bf56160 (unknown)
    @     0x7f50d4d39ada mkldnn::impl::cpu::simple_reorder_t<>
run.sh: line 5:  2581 Segmentation fault      ./python-gcc482-paddle/bin/python resnet50.py
```
# Procedure
I use pretrained model (epoch-30), and continue the training use 10x smaller learning rate.  This error occured in epoch-34.

I will try the last version of Paddle to find the problem can be reproducted or not.

**Comment by BigFishMaster** [comment]
I find the reason. 
I update the paddle version when training...
